org.cryptomator.Cryptomator.yaml

```diff
@@ -0,0 +1,100 @@
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=org.gtk.vfs
+cleanup:
+  - /include
+  - /lib/pkgconfig
+modules:
+  - name: libfuse
+    config-opts:
+      - MOUNT_FUSE_PATH=/app/bin
+    post-install:
+      - install fusermount-wrapper.sh /app/bin/fusermount
+    sources:
+      - type: archive
+        url: https://github.com/libfuse/libfuse/releases/download/fuse-2.9.9/fuse-2.9.9.tar.gz
+        sha256: d0e69d5d608cc22ff4843791ad097f554dd32540ddc9bed7638cc6fea7c1b4b5
+      - type: file
+        path: build-aux/fusermount-wrapper.sh
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk17/install.sh
+  - name: cryptomator
+    buildsystem: simple
+    build-options:
+      env:
+        PATH: /app/bin:/usr/bin:/usr/lib/sdk/openjdk17/bin
+        MAVEN_OPTS: -Dmaven.repo.local=.m2/repository
+        JAVA_HOME: /usr/lib/sdk/openjdk17/jvm/openjdk-17
+        VERSION: 1.6.5
+        REVISION_NO: '1'
+    build-commands:
+      - mvn clean package -DskipTests -Plinux
+      - cp dist/linux/launcher* target
+      - cp target/cryptomator-*.jar target/mods
+      - cd target
+      - jlink
+        --output runtime
+        --module-path "${JAVA_HOME}/jmods"
+        --add-modules java.base,java.desktop,java.logging,java.naming,java.net.http,java.scripting,java.sql,java.xml,jdk.unsupported,jdk.crypto.ec,jdk.accessibility
+        --no-header-files
+        --no-man-pages
+        --strip-debug
+        --compress=1
+      - jpackage
+        --type app-image
+        --runtime-image runtime
+        --input target/libs
+        --module-path target/mods
+        --module org.cryptomator.desktop/org.cryptomator.launcher.Cryptomator
+        --dest .
+        --name Cryptomator
+        --vendor 'Skymatic GmbH'
+        --copyright '(C) 2016 - 2022 Skymatic GmbH'
+        --java-options '-Xss5m'
+        --java-options '-Xmx256m'
+        --java-options '-Dfile.encoding='utf-8''
+        --java-options '-Dcryptomator.logDir='~/.local/share/Cryptomator/logs''
+        --java-options '-Dcryptomator.settingsPath='~/.config/Cryptomator/settings.json:~/.Cryptomator/settings.json''
+        --java-options '-Dcryptomator.ipcSocketPath='~/.config/Cryptomator/ipc.socket''
+        --java-options '-Dcryptomator.mountPointsDir='~/.local/share/Cryptomator/mnt''
+        --java-options '-Dcryptomator.showTrayIcon=false'
+        --java-options "-Dcryptomator.buildNumber='flatpak-${REVISION_NO}'"
+        --java-options "-Dcryptomator.appVersion='${VERSION}'"
+        --app-version "${VERSION}.${REVISION_NO}"
+        --verbose
+      - cp -R Cryptomator /app/
+      - ln -s /app/Cryptomator/bin/Cryptomator /app/bin/cryptomator
+      - install -D -m0644 -t /app/share/applications/ org.cryptomator.Cryptomator.desktop
+      - install -D -m0644 -t /app/share/icons/hicolor/scalable/apps/ org.cryptomator.Cryptomator.svg
+      - install -D -m0644 -t /app/share/metainfo/ org.cryptomator.Cryptomator.appdata.xml
```

Comment:
I think using `.metainfo.xml` is the way to go for this file. Would be great if you folks can generate this one at some point on your end, and the Flatpak manifest reuses it.